src/main/java/Event.java

```diff
@@ -0,0 +1,32 @@
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class Event extends Task {
+    private Date startPeriod;
+    private Date endPeriod;
+    private SimpleDateFormat formatter = new SimpleDateFormat("dd/MM/yyyy HHmm");
+
+    Event(String description, String period) throws ParseException {
```

Comment:
Personally feel its better to handle the Parsing in the Parser class, and then pass it directly to the constructor on line 17, hence keeping to the Single Responsibility Principle.